docs/02-app/02-api-reference/04-functions/unstable_cache.mdx

```diff
@@ -39,10 +39,36 @@ const data = unstable_cache(fetchData, keyParts, options)()
   - `tags`: An array of tags that can be used to control cache invalidation.
   - `revalidate`: The number of seconds after which the cache should be revalidated. Omit or pass `false` to cache indefinitely or until matching `revalidateTag()` or `revalidatePath()` methods are called.
 
+> **Tags vs KeyParts**:
+>
+> - tags: They are used for cache invalidation, NextJS does not use it to identify the function signature. 
+> - keyParts: They are used by NextJS to uniquely identify the key of the data cached.
+
 ## Returns
 
 `unstable_cache` returns a function that when invoked, returns a Promise that resolves to the cached data. If the data is not in the cache, the provided function will be invoked, and its result will be cached and returned.
 
+## Example
+
+```jsx
+import { getUser } from './data';
+import { unstable_cache } from 'next/cache';
+
+
+
+export default async function Page({ params }: { params: { userID: string } }) {
+ const getCachedUser = unstable_cache(
```

Comment:
Let's remove everything but the function call and import from this code snippet

Reply:
Well my thought process was that the example should ideally be like a real world thing, so the params from the page.. 

Just to confirm you want me to remove all the page specific code right?

Reply:
@leerob this example showcases that if you used variables that is not passed as the parameter of the unstable_cache function, it needs to be added to the keyParts.

We can make this example simpler but we would still need to communicate the gotchas to the users if a closure is used inside the function wthout passing it as arguments

Reply:
It's not "real" here regardless because `getUser` isn't a real function, so trying to minimize the amount of code shown. Let's keep it to just `getCachedUser`, which can still have the example with passing a variable like `userId`.

Reply:
Updated the function, let me know if that's what you wanted wrt the example.